packages/web/src/app/shell/page-with-sidenav/page-with-sidenav.component.html

```diff
@@ -18,8 +18,9 @@
         <div class="mobile-navbar">
           @if (currentRoute === 'brand') {
             <app-locale-picker class="e-mb-16 e-justify-content-end" />
+          } @else if (currentRoute !== 'components') {
+            <app-mobile-navbar />
```

Comment:
Is this intentional? It breaks the navbar on mobile in e.g. the brand main page

Reply:
Yes, I removed this because it adds the Components list like this to the page. And with what I have implemented it just gets added to the bottom of the page if it is not removed
<img width="665" alt="image" src="https://github.com/3lvia/designsystem/assets/111271903/01aeb8e3-359f-4720-9b0e-bde0c83b9cf2">

Reply:
There's probably something wrong with it as it is now though, as it causes there to be no menu on brand 
![image](https://github.com/3lvia/designsystem/assets/38653456/4a9d4431-4087-46c4-a316-3decbbce6954)